tests/Faithlife.Analyzers.Tests/Helpers/DiagnosticVerifier.Helper.cs

```diff
@@ -16,10 +19,18 @@ namespace Faithlife.Analyzers.Tests
 	/// </summary>
 	public abstract partial class DiagnosticVerifier
 	{
-		private static readonly MetadataReference CorlibReference = MetadataReference.CreateFromFile(typeof(object).Assembly.Location);
-		private static readonly MetadataReference SystemCoreReference = MetadataReference.CreateFromFile(typeof(Enumerable).Assembly.Location);
-		private static readonly MetadataReference CSharpSymbolsReference = MetadataReference.CreateFromFile(typeof(CSharpCompilation).Assembly.Location);
-		private static readonly MetadataReference CodeAnalysisReference = MetadataReference.CreateFromFile(typeof(Compilation).Assembly.Location);
+		private static readonly string[] s_assemblyReferences =
+		{
+			"System.Collections",
+			"System.Linq",
+			"System.Private.CoreLib",
+			"System.Runtime",
+			"Microsoft.CodeAnalysis",
+			"Microsoft.CodeAnalysis.CSharp",
+		};
+
+		private static readonly IReadOnlyList<MetadataReference> s_metadataReferences = s_assemblyReferences
+			.Select(x => (MetadataReference) MetadataReference.CreateFromFile(Assembly.Load(x).Location)).ToList();
```

Comment:
Nitpick: move these to the end of the class if we're rewriting them anyway.

Reply:
1d8982e6c7a50812adcd333ea8cd3a88ca6fdcf7